cmd/ike/session/session_test.go

```diff
@@ -0,0 +1,40 @@
+package session_test
+
+import (
+	"github.com/aslakknutsen/istio-workspace/cmd/ike/session"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Session operations", func() {
+
+	Context("route parsing", func() {
+
+		It("should return nil with no error on empty string", func() {
+			r, err := session.ParseRoute("")
+			Expect(err).ToNot(HaveOccurred())
+			Expect(r).To(BeNil())
+		})
+
+		It("should error on wrong type format", func() {
+			_, err := session.ParseRoute("a=b")
+			Expect(err).To(HaveOccurred())
```

Comment:
How about checking on the message too? As we don't have error types it might at least ensure we actually cover the expected error path.

I can take care of it if you agree.

Reply:
Sure, could do that.

Reply:
Done in 87886dd